readalongs/text/util.py

```diff
@@ -22,6 +22,8 @@
 # too many times, so define it anyway.
 unicode = str
 
+CURRENT_WEB_APP_VERSION = "1.2.0"
```

Comment:
We're actually at 1.2.1 now... it's hard to keep up!

Reply:
I will make it 1.2.x. This way we only update when we do minor releases and not for every patch